### PR TITLE
Feature/advania ark client

### PR DESCRIPTION
**Pull request by stullifugl**
# \<Description\>

Adding an ark client as asked for from Ægir
Also updating a redirect-url for HMS client since I received wrong redirect url the first time

## What

New client

### CODE REVIEW

File: libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const clients = [
+      {
+        client_id: 'island-ark-1',
+        require_client_secret: false,
+        enable_local_login: true,
+        require_pkce: true,
+        allow_offline_access: true,
```
Comment (valurefugl) [inline libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js:11]: do they need refresh tokens?

File: libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        client_id: 'island-ark-1',
+        grant_type: 'authorization_code',
+      },
+    ]
+
+    const resourceClaims = [
+      {
+        api_resource_name: 'api_resource',
+        claim_name: 'name',
+      },
```
Comment (valurefugl) [inline libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js:81]: is this the only claim they need in the payload?

File: libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js (hunk 1 above)
Comment (stullifugl) [reply libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js]: Refresh token implementation removed

File: libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js (hunk 2 above)
Comment (stullifugl) [reply libs/auth-api-lib/seeders/20201210073716-island-is-add-payment-ark-client.js]: payload put in id_token, not access_token

**Review by valurefugl — APPROVED** [review]

**Pull request merged**